index.html

```diff
@@ -261,11 +266,11 @@ <h2 class="lng_elmnt">Terminal</h2>
                 name="message"
                 placeholder="Enter text here"
                 maxlength="500"
-                required
+                id="text_arr"
               ></textarea>
             </li>
             <li>
-              <button class="butn_submit" type="submit">Get in touch</button>
+              <button class="butn_submit" type="submit" onclick="return val();">Get in touch</button>
```

Comment:
- [ ] [Optional] A return statement should only be used within a function body (Javascript). Kindly remove it. I will suggest you make use of the event listener in your javascript file.